Intallation script download-latest.sh is now working

I could not install latest version of ctx tool by the command from instruction

```
curl -sSL https://raw.githubusercontent.com/context-hub/generator/main/download-latest.sh | sh
```

I've got error:

```
Downloading the latest version
------------------------------

  >>  Preparing download from: https://github.com/context-hub/generator/releases/download/https/ctx-https-darwin-arm64


#=#=#                                                                          curl: (22) The requested URL returned error: 404

] Failed!
 [ERROR] Failed to download https://github.com/context-hub/generator/releases/download/https/ctx-https-darwin-arm64
```

seems last version detection code is broken
if set version manually script is working well

--- a/download-latest.sh
+++ b/download-latest.sh
@@ -159,8 +159,8 @@ get_latest() {
     curl -H "Authorization: token $GITHUB_PAT" -s "$latest_release" >"$temp_file" || return 1
   fi
 
-  latest="$(grep <"$temp_file" '"tag_name":' | cut -d ':' -f2 | tr -d '"' | tr -d ',' | tr -d ' ' | tr -d 'v')"
-  latestV="$(grep <"$temp_file" '"tag_name":' | cut -d ':' -f2 | tr -d '"' | tr -d ',' | tr -d ' ')"
+  latest="$(grep <"$temp_file" '"tag_name":' | sed 's/.*"tag_name"[[:space:]]*:[[:space:]]*"\([^"]*\)".*/\1/' | sed 's/^v//')"
+  latestV="$(grep <"$temp_file" '"tag_name":' | sed 's/.*"tag_name"[[:space:]]*:[[:space:]]*"\([^"]*\)".*/\1/')"
 
   rm -f "$temp_file"
 

